src/main/java/io/personium/client/DavCollection.java

```diff
@@ -269,6 +269,15 @@ public ServiceCollection service(String name) {
         return new ServiceCollection(this.accessor, UrlUtils.append(this.getPath(), name));
     }
 
+    /**
+     * Return self string.
+     * @return GET Response from this
```

Comment:
Can you adapt the comments to fit the contents of this method, along with other getString comments?